src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -141,3 +141,21 @@ export const PTSD = 'ptsd';
 export const NINE_ELEVEN = '2001-09-11';
 
 export const ERR_MSG_CSS_CLASS = '.usa-input-error-message';
+
+export const submissionStatuses = {
```

Comment:
ARE WE NOT GOING WITH CAPS LOCK FOR THESE?

Reply:
Hehehe, I thought about it. I just didn't really care for `SUBMISSION_STATUSES.PENDING` or whatever. :man_shrugging:

I'm alright with changing them. I just don't like the way it looks.